skxray/core/roi.py

```diff
@@ -531,7 +473,6 @@ def roi_kymograph(images, labels, num):
     """
     roi_kymo = []
     for n, img in enumerate(images):
-        roi_kymo.append((roi_pixel_values(img,
-                                          labels == num)[0])[0])
+        roi_kymo.append((roi_pixel_values(img, labels == num)[0])[0])
```

Comment:
The first zero is because `roi_pixel_values` returns two values: the list of pixels in each roi and the list of roi indices.  The first `[0]` is to grab the list of pixels in each roi and the second `[0]` is to grab the first and only element out of that list of pixels.  I agree that this is pretty ugly. How should that roi_pixel_values API be changed?